agent/lib_zend_http.c

```diff
@@ -19,6 +19,23 @@ typedef enum _nr_zend_http_adapter {
   NR_ZEND_ADAPTER_OTHER = 1,
 } nr_zend_http_adapter;
 
+/*
+ * Laminas is a rebranding of Zend, but the logic remains the same,
+ * it is simply a name change and corresponds directly to Zend 3.x.
+ *
+ * Refer to the manual for up to date specs:
+ * https://docs.laminas.dev/
+ *
+ * In order to toggle between `zend` and `laminas` naming schemes, the names
+ * are stored in defines.  The default is the `zend` naming scheme.
+ * If `laminas` is detected, the values are redefined.
+ */
+static char library_name[50] = "Zend";
+static char curl_adapter_typename[50] = "Zend_Http_Client_Adapter_Curl";
+static char uri_http_typename[50] = "Zend_Uri_Http";
+static char http_client[50] = "Zend_Http_Client";
+static char http_client_request[50] = "Zend_Http_Client::request";
```

Comment:
I wonder if we can do this by freeing and allocating new memory for the laminas naming scheme in `nr_laminas_http_enable()`, instead of copying the strings over?